p2p/kademlia/dht.go

```diff
@@ -89,7 +93,7 @@ func NewDHT(ctx context.Context, store dao.Key, options *Options) (*DHT, error)
 	dht.ht = ht
 	dht.networking = &realNetworking{}
 
-	if err = store.Init(ctx); err != nil {
+	if err = store.Init(ctx, options.DataDir); err != nil {
```

Comment:
I think sqlite needs to get a file, not a directory.
So, I think you need to add something like

```go
const dbFilename = "p2p.sqlite"
......
filepath.Join(options.DataDir, dbFilename)
```